components/react/authSignerModal.tsx

```
@@ -280,7 +273,14 @@ const SignModal = ({
             />
             <h3 className="text-xl font-semibold">Approve transaction?</h3>
           </div>
-          <button className="btn btn-sm btn-circle btn-ghost" onClick={onClose}>
+          <button
+            id={'close-modal'}
+            className="btn btn-sm btn-circle btn-ghost"
+            onClick={() => {
+              debugger;
```

Biome (1.9.4)

[error] This is an unexpected use of the debugger statement.

Unsafe fix: Remove debugger statement


(lint/suspicious/noDebugger)